decidim-accountability/app/commands/decidim/accountability/admin/create_timeline_entry.rb

```diff
@@ -5,39 +5,12 @@ module Accountability
     module Admin
       # This command is executed when the user creates a TimelineEntry
       # for a Result from the admin panel.
-      class CreateTimelineEntry < Decidim::Command
-        def initialize(form, user)
-          @form = form
-          @user = user
-        end
+      class CreateTimelineEntry < Decidim::Commands::CreateResource
+        fetch_form_attributes :decidim_accountability_result_id, :entry_date, :title, :description
 
-        # Creates the timeline_entry if valid.
-        #
-        # Broadcasts :ok if successful, :invalid otherwise.
-        def call
-          return broadcast(:invalid) if @form.invalid?
+        protected
```

Comment:
Shouldn't this be private? 

```suggestion
        private
```

By using protected I was expecting that this class (`CreateTimelineEntry`) is inherited somewhere... Is that the case?

Reply:
addressed in 29052b5d02